### DISCUSSION TITLE
annoying (unintended?) cooldown on the missions-giving npc in singleplayer

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#15031] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

talking to the guy that gives you missions has a drastic cooldown that slows down pace, and it didn't use to be there. i'm not that impatient but the cooldown is like 5 seconds inbetween each mission which gets annoying when you're trying to leave the station

### Reproduction steps

try to talk to the guy
get mission
takes 10 seconds to talk again

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.7.0.1 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in the unstable build, and the fix will be included in the upcoming release. Closing
